pachyderm/base/params.env

```diff
@@ -0,0 +1,12 @@
+PACH_ROOT=/pach
+NUM_SHARDS=16
+EXPORT_OBJECT_API=false
+WORKER_IMAGE=pachyderm/worker:1.8.6
+WORKER_SIDERCAR_IMAGE=pachyderm/pachd:1.8.6
+WORKER_IMAGE_PULL_POLICY=IfNotPresent
+PACHD_VERSION=1.8.6
+METRICS=true
+LOG_LEVEL=info
+BLOCK_CACHE_BYTES=0G
+PACHYDERM_AUTHENTICATION_DISABLED_FOR_TESTING=false
+ETCD_SERVICE_PORT=2379
```

Comment:
Are these parameters being used? I am not able to find their usage in any files.

Reply:
they are used by the configmap generator in the base kustomization.yaml file